doit/cmd_resetdep.py

```diff
@@ -0,0 +1,50 @@
+from .cmd_base import DoitCmdBase, check_tasks_exist
+from .cmd_base import subtasks_iter
+
+
+class ResetDep(DoitCmdBase):
+    name = "reset-dep"
+    doc_purpose = "recompute the state of file dependencies"
+    doc_usage = "[TASK ...]"
+    doc_description = None
```

Comment:
Please add a long description mentioning in which situation this command is useful.